<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoClipDialog</class>
 <widget class="QDialog" name="VideoClipDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Clip Player</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QVBoxLayout" name="videoLayout">
     <item>
      <widget class="QLabel" name="ffmpegLabel">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="positionSlider">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="timeLayout">
     <item>
      <widget class="QLabel" name="currentTimeLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalTimeLabel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="playPauseButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgb(243,115,33);
    background: none;
    border: 1px solid rgb(243,115,33);
    border-radius: 15px;  /* 테두리 둥글게 */
    font: 500 9pt 'Quicksand Medium';
    icon-size: 24px;  /* 아이콘 크기 증가 */
    padding: 5px;  /* 버튼 내부 여백 */
}
QPushButton:hover {
    color: white;
    background-color: rgba(243,115,33,0.7);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../images.qrc">
         <normaloff>:/images/play2.png</normaloff>:/images/play2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    color: rgb(243,115,33);
    background: none;
    border: 1px solid rgb(243,115,33);
    border-radius: 15px;  /* 테두리 둥글게 */
    font: 500 9pt 'Quicksand Medium';
    icon-size: 24px;  /* 아이콘 크기 증가 */
    padding: 5px;  /* 버튼 내부 여백 */
}
QPushButton:hover {
    color: white;
    background-color: rgba(243,115,33,0.7);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../images.qrc">
         <normaloff>:/images/stop2.png</normaloff>:/images/stop2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
